pkg/cmd/controller/metrics.go

```diff
@@ -0,0 +1,19 @@
+package controller
+
+import (
+	"context"
+)
+
+// AuthenticatorMetrics specifies a set of methods that are used to register various metrics
+type AuthenticatorMetrics struct {
```

Comment:
Do we need to create an interface here? Can't we just add a godoc that this implements the `webhooktoken.AuthenticatorMetrics` interface